src/layout/boxmodel.ts

```diff
@@ -11,23 +12,23 @@ let timeout: number = null;
 
 export function compute(): void {
     clearTimeout(timeout);
-    timeout = window.setTimeout(schedule, config.lookahead);
+    timeout = setTimeout(schedule, config.lookahead);
 }
 
 function schedule(): void {
     task.schedule(boxmodel);
 }
 
 async function boxmodel(): Promise<void> {
-    let timer = Metric.BoxModelTime;
+    let timer = Metric.BoxModelCost;
```

Comment:
>BoxModelCost [](start = 23, length = 12)

Should come up with a naming convention for metric ids including: 

* xxxCount coutns something
* xxxDuration. Counts the time of something in known units (e.g., milliseconds)
* xxxTime. The time of something measured on the performance now timeline in ms.
* xxxTimestamp. Full epoch timestamp value (not used)?
* xxxByteCount of xxxBytes or xxxSize. Number of bytes in something.

Reply:
Good point. I will follow these conventions. Thanks.

Reply:
Followed this convention. Except for cost, I used Latency as the suffix: xxxLatency: (ms)